lua/telescope/config.lua

```diff
@@ -108,6 +108,7 @@ local layout_config_defaults = {
   bottom_pane = {
     height = 25,
     prompt_position = "top",
+    preview_cutoff = 40,
```

Comment:
I think that 120 is a better default value here, as it matches the behaviour of the most similar other layout, `horizontal`.